source/game/field/ObjectDirector.cc

```diff
@@ -28,7 +28,7 @@ ObjectDirector *ObjectDirector::Instance() {
 }
 
 /// @addr{0x8082A38C}
-ObjectDirector::ObjectDirector() = default;
+ObjectDirector::ObjectDirector() : m_flowTable("ObjFlow.bin") {}
```

Comment:
`ObjectFlowTable` ctor is only ever called from here, which makes me question if we even want to manually specify `ObjFlow.bin`. Maybe just make the filename a `constexpr` in the `ObjectFlowTable` ctor?

Reply:
I was conflicted about this, so I defaulted to the base game's implementation.